tests/conftest.py

```diff
@@ -135,3 +135,20 @@ def atomic_batch(nequip_dataset):
 
 # Use debug mode
 set_irreps_debug(True)
+
+
+@pytest.fixture(scope="function")
+def per_species_set():
+    dtype = torch.get_default_dtype()
+    torch.manual_seed(0)
```

Comment:
```suggestion
    rng = torch.Generator().manual_seed(0)
```

Reply:
I want to start avoiding setting too much global state in tests to avoid weird race conditions where they pass when parallelized over 2 cores (CI) but not more (local)